colleges_and_buildings.md

```diff
@@ -321,9 +321,12 @@ Biomedical Engineering，Physiology和Pharmacology的主场，
 CBS微生物学的实验室也位于Moos Tower地下。
 
 **Diehl Hall**  
-Bio-Medical Library 所在地，通过地下通道与 Health Area 的其他建筑相连。
-（因此其实是可以通过Moos Tower的地下通道入口直接走到Bio-Medical Library的）
+神经科学/精神病学与儿科的主场，通过地下通道与 Health Area 的其他建筑相连。
+（因此其实是可以通过Moos Tower的地下通道入口直接走到Diehl Hall的）
 
+**Health Sciences Education Center**
+Health Sciences Library 的所在地（原Bio-Medical Library的新地址, 该图书馆提供自习或者小组学习的场所。
```

Comment:
missing closing parenthesis

Reply:
The building itself is also connected to other bldgs in the Health Area via Tunnels